core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -397,45 +402,27 @@ private void configPeerNodes(@Nullable Socks5Proxy proxy) {
     // Peer configuration check
     ///////////////////////////////////////////////////////////////////////////////////////////
 
-    private static ListenableFuture<Void> checkPeerConfiguration(Peer peer) {
-      var versionMessageFuture = getVersionMessage(peer);
-
-      Function<VersionMessage,Void> handler =
-        (versionMessage) -> handleConfiguration(peer, versionMessage);
-
-      return Futures.transform(versionMessageFuture, handler);
-    }
-
-    private static Void handleConfiguration(Peer peer, VersionMessage versionMessage) {
+    /* Provided a Peer and its VersionMessage, will check and handle Peer's advertised
+     * configuration. The VersionMessage is provided separately, because it is not
+     * always available and this is meant to be called from a listener that is triggered
+     * when the VersionMessage becomes available.
+     */
+    private Void handleConfiguration(Peer peer, VersionMessage versionMessage) {
```

Comment:
The uppercase Void is probably a leftover from the previous code. Should be `void`.